opensrp-reporting/src/main/java/org/smartregister/reporting/repository/DailyIndicatorCountRepository.java

```diff
@@ -292,6 +389,38 @@ public static void addValueSetColumns(@NonNull SQLiteDatabase database) {
 
     public static void aggregateDailyTallies(@NonNull SQLiteDatabase database) {
         // Code to migrate the code over from incremental tallies should be written here
```

Comment:
So with this implementation, does it mean https://github.com/OpenSRP/opensrp-client-reporting/blob/7257fb5d253ad1d23c9e8a0f5eb46d94a942ec24/opensrp-reporting/src/main/java/org/smartregister/reporting/repository/IndicatorQueryRepository.java#L171-L173 should be removed/deleted?